src/features/parsers/parseMidi.ts

```diff
@@ -29,7 +29,8 @@ export default function parseMidi(midiData: ArrayBufferLike): Song {
         i,
         {
           name: track.name,
```

Comment:
Sounds good to me! I don't think there should be any negative re-`percussions`, ha!

Reply:
Oh, we could use the same approach for left/right hand for drums also. In case a teacher wants to do that. That's actually how Melodics work. (https://melodics.com/drums) They have a different track for left and right.